tox.ini

```diff
@@ -5,7 +5,7 @@ isolated_build = true
 
 [testenv]
 deps =
-    -requirements-tests.txt
+    -rrequirements-tests.txt
```

Comment:
@mfldavidson I know you removed the extra `r` in https://github.com/prschmid/zoomus/commit/d16dc043d1d27aa638db262e75ead7f407af93a8, but looks like this isn't a typo and travis fails.  See this failing build (and then the subsequent fix). https://travis-ci.org/github/prschmid/zoomus/jobs/764225842